eng/pipelines/templates/steps/ref-updater.yml

```diff
@@ -8,28 +8,41 @@ parameters:
 steps:
 - ${{ each repo in parameters.Repos }}:
   - pwsh: |
-      git clone https://github.com/azure/${{ repo }}
-    displayName: Clone ${{ repo }}
+      if (!(Test-Path ${{ repo.key }}))
+      {
+        git clone https://github.com/azure/${{ repo.key }}
+        if ("${{ repo.value.branch }}")
+        { 
+          Push-Location ${{ repo.key }}
+          Write-Host "git checkout -b ${{ repo.value.branch }} origin/${{ repo.value.branch }}"
+          git checkout ${{ repo.value.branch }}
```

Comment:
This command doesn't match what you are writing to the log. Which one is wrong?

Reply:
Thanks the logging is from a previous iteration. I will update it.